db/schema.rb

```diff
@@ -50,17 +50,16 @@
     t.boolean "call_present"
     t.date "call_open_date"
     t.date "call_close_date"
-    t.string "intended_beneficiaries", array: true
     t.string "roda_identifier_fragment"
     t.string "roda_identifier_compound"
+    t.string "intended_beneficiaries", array: true
```

Comment:
Did you mean to commit these unrelated `db/schema.rb` changes?

Reply:
Whoops, no!